test/Gridify.Tests/GridifyMapperShould.cs

```diff
@@ -149,8 +149,31 @@ public void AddMap_DuplicateKey_ShouldThrowErrorIfOverrideIfExistsIsFalse()
       //The thrown exception can be used for even more detailed assertions.
       Assert.Equal("Duplicate Key. the 'test' key already exists", exception.Message);
    }
+   
+   [Theory]
+   [InlineData(typeof(DateTime), 0)]
```

Comment:
one improvement would be to add a test case that has a DateTime ... false test is not so useful

Reply:
one other thing that I just remembered is nested mappings... does it work with nested ones? plus, mappings that has a `Select` method for collections. we need to test more before merging it